android/src/main/java/com/doublesymmetry/trackplayer/module/MusicModule.kt

```diff
@@ -110,7 +148,7 @@ class MusicModule(reactContext: ReactApplicationContext) : ReactContextBaseJavaM
         constants["STATE_PAUSED"] = State.Paused.state
         constants["STATE_STOPPED"] = State.Stopped.state
         constants["STATE_BUFFERING"] = State.Buffering.state
-        constants["STATE_CONNECTING"] = State.Connecting.state
+        constants["STATE_LOADING"] = State.Loading.state
```

Comment:
Should the Error state be here too?

Reply:
Yes it would need to be added – but actually we aren't using these constants anywhere anymore.. Perhaps they can be removed? @dcvz do you have thoughts?